### PR TITLE
Implement May Warbond files.

### LINKED ISSUE #38: [FEATURE]: Polar Patriots Warbond Release Implementation
### What problem is solved?

Will add the undocumented warbond released 9 May 2024

### What is the solution?

Generate new warbond_name.json and update warbonds.json to include new warbond.

### What value does this add?

Proper tracking of all warbonds

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by chatterchats**
fixes #38

### CODE REVIEW

File: items/armor/armor.json [hunk 1]
```diff
@@ -709,6 +769,56 @@
...
+		"type": 1,
+		"slot": 0,
+		"armor_rating": 100,
+		"speed": 100,
+		"stamina_regen": 100,
+		"passive": 0
+	},
+	"4282790522": {
+		"name": "Dissident's Nightmare",
+		"description": "",
```
Comment (dealloc) [inline items/armor/armor.json:814]: no description on this one?

File: items/armor/armor.json [hunk 2]
```diff
@@ -1349,6 +1459,26 @@
 		"stamina_regen": 100,
 		"passive": 0
 	},
+	"3892781112": {
+		"name": "Order of the Venerated Ballot",
+		"description": "",
```
Comment (dealloc) [inline items/armor/armor.json:1464]: no description?

File: items/armor/armor.json (hunk 1 above)
Comment (chatterchats) [reply items/armor/armor.json:814]: Yee, haven't got any descriptions from crowd sourcing and don't got the medals for it myself to see

**Comment by chatterchats** [comment]
@dealloc Ready for Review

File: items/item_names.json
Comment (dealloc) [inline items/item_names.json:1]: is it normal mix_id's have been changed? (eg. Peacemaker and G-12)

**Comment by chatterchats** [comment]
In this case yes, messed up the way I built item_names so I took the time to rebuild it better, side effect is those 2 changed up more then thry normally would

File: items/item_names.json
Comment (chatterchats) [reply items/item_names.json:1]: In this case yes, messed up the way I built item_names so I took the time to rebuild it better, side effect is those 2 changed up more then thry normally would

Comment (dealloc) [reply items/item_names.json:1]: if the change is intentional, all good by me

File: items/weapons/secondary.json [hunk 3]
```diff
@@ -75,5 +75,18 @@
...
 		]
+	},
+	"1938200223": {
+		"name": "P-113 Verdict",
+		"description": "The gas-operated, semiautomatic P-113 chambers the 14mm 'Rapid Deliberation,' the largest centerfire cartridge of any magazine-fed pistol.",
+		"damage": 125,
+		"capacity": 10,
+		"recoil": 40,
+		"fire_rate": 450,
+		"fire_mode": [],
```
Comment (dealloc) [inline items/weapons/secondary.json:86]: no fire modes for this one?

Comment (chatterchats) [reply items/weapons/secondary.json]: Resolved in [f3605f2](https://github.com/helldivers-2/json/pull/41/commits/f3605f27973a3c068a3c69e8c4a72f9d4a223a79)

File: items/weapons/grenades.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-	"1": {
+	"3": {
```
Comment (dealloc) [inline items/weapons/grenades.json:2]: are these numbers supposed to change (again)?

File: items/weapons/grenades.json [hunk 5]
```diff
@@ -63,4 +63,4 @@
 		"outer_radius": 7,
 		"fuse_time": 0
 	}
-}
+}
```
Comment (dealloc) [inline items/weapons/grenades.json:66]: trailing spaces :)

File: items/weapons/secondary.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
-	"1": {
+	"2": {
```
Comment (dealloc) [inline items/weapons/secondary.json:2]: number change intentional?

**Review by dealloc — CHANGES_REQUESTED** [review]

File: items/weapons/secondary.json (hunk 6 above)
Comment (chatterchats) [reply items/weapons/secondary.json:2]: yee

File: items/weapons/grenades.json (hunk 4 above)
Comment (chatterchats) [reply items/weapons/grenades.json:2]: yee

File: items/weapons/grenades.json (hunk 5 above)
Comment (chatterchats) [reply items/weapons/grenades.json]: resolved in latest commit

**Review by dealloc — APPROVED** [review]

**Pull request merged**